kolibri/core/assets/src/core-app/apiSpec.js

```diff
@@ -101,6 +101,7 @@ import CatchErrors from '../utils/CatchErrors';
 import UiIconButton from '../views/KeenUiIconButton.vue';
 import UiToolbar from '../views/KeenUiToolbar.vue';
 import shuffled from '../utils/shuffled';
+import appCapabilities from '../utils/appCapabilities';
```

Comment:
appCapabilities is not exporting a `default` function or const, thus this is not working

Reply:
Ah yep, I changed that in response to review comments. Am reverting back to having a default based on review above.